### PR TITLE
#66 fix package.json error

### PR DESCRIPTION
**Pull request by dubbya**
package.json might not be in the local project folders when used global

### CODE REVIEW

File: lib/reporter.js [hunk 1]
```diff
@@ -12,7 +12,14 @@ var generateReport = function (options) {
...
     var featureOutput = jsonFile.readFileSync(options.jsonFile);
     var packageJsonPath = searchFileUp('package.json');
-    var packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    var packageJson = {}
+    try {
+        packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    } catch (err) {
+        console.warn("No package.json file found in: " + packageJsonPath + ", using default name and version.")
+        packageJson.name = "default"
+        packageJson.version = "0.0.0"
```
Comment (gkushang) [inline lib/reporter.js]: Can you replace the double quotes `"` with single quote above?

**Comment by dubbya** [comment]
@gkushang all done

File: lib/reporter.js [hunk 2]
```diff
@@ -12,7 +12,14 @@ var generateReport = function (options) {
...
     var featureOutput = jsonFile.readFileSync(options.jsonFile);
     var packageJsonPath = searchFileUp('package.json');
-    var packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    var packageJson = {}
+    try {
+        packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    } catch (err) {
+        console.warn('No package.json file found in: ' + packageJsonPath + ', using default name and version.');
+        packageJson.name = 'default';
+        packageJson.version = '0.0.0';
```
Comment (gkushang) [inline lib/reporter.js]: IMO - let's keep these key-pairs empty and log the warning. Will update before publishing. Thanks for pushing the commits.

**Comment by dubbya** [comment]
We will need to put more error handling further on if they are empty. I will see if i get some more time next week and i will add to the PR.  This was just to get it working for now because i needed it.

**Pull request merged**